stock_vertical_lift/models/vertical_lift_operation_inventory.py

```diff
@@ -0,0 +1,250 @@
+            ("save", "Save"),
+            # no need for release and save button here?
+            # ("release", "Release"),
+        ]
+
+    def _transitions(self):
+        return (
+            self.Transition(
+                "noop",
+                "quantity",
+                # transition only if inventory lines are found
+                lambda self: self.select_next_inventory_line(),
+            ),
+            self.Transition(
+                "quantity",
+                "save",
+                lambda self: self._has_identical_quantity(),
+            ),
+            self.Transition(
+                "quantity",
+                "confirm_wrong_quantity",
+                lambda self: self._start_confirm_wrong_quantity(),
+            ),
+            self.Transition(
+                "confirm_wrong_quantity",
+                "save",
+                lambda self: self.quantity_input == self.last_quantity_input,
+            ),
+            # if the confirmation of the quantity is different, cycle back to
+            # the 'quantity' step
+            self.Transition(
+                "confirm_wrong_quantity",
+                "quantity",
+                lambda self: self._go_back_to_quantity_input(),
+            ),
+            # go to quantity if we have lines in queue, otherwise, go to noop
+            self.Transition(
+                "save",
+                "quantity",
+                lambda self: self.process_current()
+                and self.select_next_inventory_line(),
+                # when we reach 'save', this transition is directly
+                # evaluated
+                direct_eval=True,
+            ),
+            self.Transition(
+                "save",
+                "noop",
+                lambda self: self.process_current()
+                and self.clear_current_inventory_line(),
+                # when we reach 'save', this transition is directly
+                # evaluated
+                direct_eval=True,
+            ),
+        )
+
+    quant_id = fields.Many2one("stock.quant", string="Current Quant")
+
+    quantity_input = fields.Float()
+    # if the quantity is wrong, user has to write 2 times
+    # the same quantity to really confirm it's correct
+    last_quantity_input = fields.Float()
+
+    tray_location_id = fields.Many2one(
+        comodel_name="stock.location",
+        compute="_compute_tray_data",
+        string="Tray Location",
+    )
+    tray_name = fields.Char(compute="_compute_tray_data")
+    tray_type_id = fields.Many2one(
+        comodel_name="stock.location.tray.type",
+        compute="_compute_tray_data",
+        string="Tray Type",
+    )
+    tray_type_code = fields.Char(compute="_compute_tray_data", string="Tray Code")
+    tray_x = fields.Integer(string="X", compute="_compute_tray_data")
+    tray_y = fields.Integer(string="Y", compute="_compute_tray_data")
+    tray_matrix = Serialized(string="Cells", compute="_compute_tray_data")
```

Comment:
IMO we can just use the `Json` field and drop the `base_sparse_field` dependency